lld/ELF/Relocations.cpp

```diff
@@ -504,8 +504,7 @@ int64_t RelocationScanner::computeMipsAddend(const RelTy &rel, RelExpr expr,
 }
 
 // Custom error message if Sym is defined in a discarded section.
-template <class ELFT>
-static std::string maybeReportDiscarded(Undefined &sym) {
+template <class ELFT> static std::string maybeReportDiscarded(Undefined &sym) {
```

Comment:
Looks like some clang-format changes to unchanged code have made it into the patch. If these are still worth doing can you split these up into a separate [NFC] change mentioning clang-format